dotnet-nunit/SauceLabs.Visual.Example/SauceDemo.cs

```diff
@@ -68,7 +69,41 @@ public async Task SauceDemo_CheckInventory_ShouldAddBackpackToCart()
             });
 
         var results = await VisualClient.VisualResults();
-        Assert.AreEqual(2, results?[DiffStatus.Unapproved]);
+        Assert.AreEqual(2, results?.Values.Sum());

```

Comment:
wait why? Why are we summing values?

Reply:
https://saucelabs.slack.com/archives/C051AKF347Q/p1712152749156699?thread_ts=1712152598.611539&cid=C051AKF347Q

Reply:
I'm ok with this!